packages/preview/starter-journal-article/0.2.0/template/main.typ

```diff
@@ -0,0 +1,38 @@
+#import "@preview/starter-journal-article:0.2.0": article, author-meta
+
+#show: article.with(
+    title: "Article Title",
```

Comment:
There's one unnecessary extra level of indent here.

Reply:
Thanks. The extra indent has been removed in cee5160deee5f229c2755c49dc4fa39cf62ae991 .